core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Ellipse2afp.java

```diff
@@ -56,6 +56,11 @@
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends RectangularShape2afp<ST, IT, IE, P, V, B> {
 
+    final String ELLIPSE_WIDTH_POSITIVE_OR_ZERO = "Ellipse width must be positive or zero";
```

Comment:
The errors seems to be given by the Checkstyle plugin that was now used for validating the code. Please fix the code according to the recommendations of Checkstyle, in addition to remove the IntelliJ configuration files.